pack/commands.cfg

```diff
@@ -0,0 +1,25 @@
+# -----------------------------------------------------------------
+#
+#      OpenStack standard check
+#      Need  https://github.com/kedark3/check-osp
+#
+# -----------------------------------------------------------------
+
+
+## Check commands for openstack
+define command{
+       command_name     check_osp_vm_count
+       command_line     $PLUGINSDIR$/check_osp_main.py -R $HOSTALIAS$ -u $_HOSTOSP_USER$ -p $_HOSTOSP_PASSWORD$ -t $_HOSTOSP_TENANT$ -m vm_count -c $_HOSTOSP_VM_COUNT_CRIT$ -w $_HOSTOSP_VM_COUNT_WARN$
+}
+define command{
+       command_name     check_osp_keypair_count
+       command_line     $PLUGINSDIR$/check_osp_main.py -R $HOSTALIAS$ -u $_HOSTOSP_USER$ -p $_HOSTOSP_PASSWORD$ -t $_HOSTOSP_TENANT$ -m keypair_count -c $_HOSTOSP_LOCKED_DISKS_CRIT$ -w $_HOSTOSP_LOCKED_DISKS_WARN$
+}
+define command{
+       command_name     check_osp_volumes_count
```

Comment:
To maintain consistency with the other count checks, I think `volume` should be singular, i.e. `check_osp_volume_count`